### PR TITLE
{chem}[foss/2018a] AmberTools v17

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

edit: depends on ~~#6026~~ (netCDF-Fortran) and ~~#6359~~ (tcsh)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/1a367fa154732348c1cbf96960a6f102 for a full test report.

**Comment by boegel** [comment]
```
Error: /bin/csh not found on your system! Install csh or tcsh and rerun
```

@verdurin So, we should include `tcsh` as a build dep?

**Comment by verdurin** [comment]
@boegel seems like it. Can't remember whether you need `tsch` to run it as well.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2604.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ac66a7e3c706f6561e4b2bf4da597681 for a full test report.

**Comment by boegel** [comment]
Hmm, seems like this needs similar stuff as in #5654?

For some reason that's not needed when building with the `intel` toolchain though... :-/

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/c2fedb38bcc313a083666f86fe120724 for a full test report.

File: easybuild/easyconfigs/a/AmberTools/AmberTools-17-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+description = """AmberTools consists of several independently developed packages that work well by themselves,
+ and with Amber itself. The suite can also be used to carry out complete molecular dynamics simulations,
+ with either explicit water or generalized Born solvent models."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+# download requires registration, see http://ambermd.org/AmberTools17-get.html
+sources = ['AmberTools%(version)s.tar.bz2']
+patches = ['Amber-16_fix-hardcoding.patch']
```
Comment (akesandgren) [inline easybuild/easyconfigs/a/AmberTools/AmberTools-17-foss-2018a.eb]: Should probably add all the AT-17 patches from Amber-16 in a/Amber/

That should take care of a whole bunch of problems

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 2 out of 3 (1 easyconfigs in this PR)
b-cn1519.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz, Python 2.7.12
See https://gist.github.com/5a491a93ac60369e411ff2684cc47860 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-cn1519.hpc2n.umu.se - Linux ubuntu 16.04, x86_64, Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz, Python 2.7.12
See https://gist.github.com/6c13fbcb143a718d71bfbd2727e566f7 for a full test report.

**Review by akesandgren — APPROVED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 5 out of 6 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.8.2003, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/27114d3f9bbc66ffc086e7dced1b0b12 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node2689.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/6515e4dcd8422385bd35a303b356a695 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2689.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/1dc2792c3d179c746cd345ea81f644e5 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**